packages/neon/neon/lib/src/theme/color_scheme.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'package:neon/src/theme/colors.dart';
+import 'package:neon/src/theme/neon.dart';
+
+/// A ColorScheme used in the [NeonTheme].
+@immutable
+class NeonColorScheme {
+  const NeonColorScheme({
+    this.primary = NcColors.primary,
+    this.oledBackground = Colors.black,
+  });
+
+  /// Primary color used throughout the app.
+  ///
+  /// See [ColorScheme.primary]
+  final Color primary;
+
+  /// The [ColorScheme.background] color used on OLED devices.
+  ///
+  /// This color is only used at the users discretion.
+  final Color oledBackground;
```

Comment:
Does it really make sense to have it configurable? If it is only meant for OLED theme then pitch black is the only accepted value

Reply:
Yeah that's probably right.

I moved it into there before I came up with the `NeonColors`.